nes_bnf_komentarz.md

```diff
@@ -0,0 +1,50 @@
+<meta http-equiv='Content-Type' content='text/html; charset=utf-8' />
+<h2>
+<p align="center">Zapis propozycji nowej budowy metadanych  
+Niezbędnych Elementów Struktury (NES)  
+dokumentów elektronicznych,  
+w Notacji Backusa-Naura (BNF)</p>
+</h2>
+
+<p align="right"><em>Na skróty: dokument HTML jest tu: <a href="https://stas53.github.io/tests/nes_bnf.html"> nes_bnf.html</a>
```

Comment:
Trzeba by opracować jakąś kanoniczną metodę wyświetlania dokumentów HTML, bo odnośnik do wersji generowanej z prywatnego repo (gdzie w dodatku wersja może być niezsynchronizowana) to raczej rozwiązanie ad-hoc. Może to dobra okazja, żeby powiedzieć, że niniejsze repozytorium zawiera kod źródłowy analiz, a wersja skompilowana jest widoczna gdzie indziej, np. wystawiana przez Github Pages z wykorzystaniem Jekylla? (Przy okazji można by dodać do układu strony logotypy itp.).